certified-connectors/App Power Forms/apiDefinition.swagger.json

```diff
@@ -0,0 +1,647 @@
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/AddAdaptiveCard": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "x-ms-dynamic-schema": {
+                "operationId": "AddCardSchema",
+                "parameters": {
+                  "cardData": {
+                    "parameter": "card"
+                  },
+                  "cardName": {
+                    "parameter": "name"
+                  }
+                }
+              },
+              "x-ms-dynamic-properties": {
+                "operationId": "AddCardSchema",
+                "parameters": {
+                  "cardData": {
+                    "parameterReference": "body/card"
+                  },
+                  "cardName": {
+                    "parameterReference": "body/name"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "Create new form from Adaptive Card",        
+        "operationId": "AddAdaptiveCard",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "required":["name", "card"],
+              "properties": {
+                "name": {
+                  "type": "string",
+                  "title": "Card name"
+                },
+                "card": {
+                  "type": "string",
+                  "title": "Adaptive Card JSON"
+                },
+                "cardAfterSubmit": {
+                  "type": "string",
+                  "title": "Card displayed after submit"
+                }
+              }
+            }
+          }
+        ],
+        "description": "Use this action to create a form based on Adaptive Card JSON."
+      }
+    },
+    "/AddCardSchema/{cardName}": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "No need to specify response schema",
+            "schema":{}
+          }
+        },
+        "operationId": "AddCardSchema",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "name": "cardData",
+            "in": "body",
+            "required": false,
+            "schema": {}
+          },
+          {
+            "name": "cardName",
+            "in": "path",
+            "type": "string",
+            "x-ms-url-encoding": "single",
+            "required": true
+          }
+        ]
+      }
+    },
+    "/AddCardFieldSchema/{formName}/{fieldName}": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "No need to specify response schema",
+            "schema":{}
```

Comment:
@TerhoAntila Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Reply:
Resolution committed.